### PR TITLE
fix: Cypress fix - failed embedsettings spec

### PR DESCRIPTION
**Pull request by dipyamanbiswas07**
## Description

Fixes test case failures in EmbedSettings spec.
## Type of change

> Please delete options that are not relevant.

- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts [hunk 1]
```diff
@@ -22,11 +22,8 @@ describe("In-app embed settings", () => {
...
         _.inviteModal.locators._shareSettingsButton,
       );
-      _.agHelper.AssertElementExist(_.inviteModal.locators._upgradeButton);
-      cy.get(_.inviteModal.locators._upgradeButton).should(
-        "have.attr",
-        "href",
-        "https://customer.appsmith.com/plans",
+      cy.get(_.inviteModal.locators._upgradeContent).contains(
+        "Appsmith Business Edition",
       );
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts:27]: Use agHelper.GetNAssertContains() insetad of direct statements in spec files - thus adhering to TS style of scripting.

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts [hunk 2]
```diff
@@ -40,11 +37,8 @@ describe("In-app embed settings", () => {
...
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts:42]: Same as above.

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts (hunk 1 above)
Comment (dipyamanbiswas07) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts]: Done

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts (hunk 2 above)
Comment (dipyamanbiswas07) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/SettingsPane/EmbedSettings_spec.ts]: Done

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**